### ISSUE TITLE
Bug Macs Fan Control 1.5.8

### ISSUE THREAD

**Issue by YJ5754** [crystalidea/macs-fan-control#403]
Hello! I have just updated Macs Fan Control (MFC) to the proposed 1.5.8 version. Since then, MFC is unvisible and unaccessible. Nothing in the menu bar. When I click on the application, a message occurs saying the application is already running. When I try to "force to quit" MFC from apple menu, the app does not appear (only Finder). When I try to drag the application to the bin, a message occurs saying it is impossible since the app is running. In summary, MFC 1.5.8 is just a ghost.
I tried to reload the previous version 1.5.7 but when I unzipped it, i get the so-called running 1.5.8 version screen...
Does anyone has the same issue? How to fix it?
Many thanks for your help.
Kind regards,

Yves (France)

Macbook Pro 15" Retina mid-2015
Mac OS 10.12.6

**Comment by kleuter** [comment]
Hi,

Please run Activity Monitor and force kill the app process

**Comment by YJ5754** [comment]
Hi,
It works, many thanks. I ran the following procedure:
I founded Activity Monitor and landed in a somewhat unknown territory… But in the upper bar, I selected “Memory” and found the Mac Fan Control app in the column “Operations name” on the left side of the window. I selected it and clicked on the far upper left button displaying an “x”. A window opened asking me if I wanted to delete this application. I confirmed and the app disappeared from the list.
Then I went to the Applications folder on my mac. The Macs Fan Control app was always present in the folder but it was possible to drag it to the trash and then empty the latter.
Hope this helps
Kind regards
Yves
